modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -412,7 +416,9 @@
  * @since JavaFX 2.0
  */
 @IDProperty("id")
-public abstract class Node implements EventTarget, Styleable {
+public abstract sealed class Node
+        implements EventTarget, Styleable
+        permits AbstractNode, Camera, LightBase, Parent, SubScene, Canvas, ImageView, Shape, Shape3D {
```

Comment:
We may need to use stronger wording in javadoc in regards to extending Nodes.  should -> must, etc.  (or, rather, 'cannot' and possibly explain why?)

Reply:
I've removed the language that you can't extend those classes, as it is now redundant because it is enforced by the compiler. We shouldn't document obvious facts without providing any added value (for example, adding context and reasoning _why_ it's the way it is).